tsg/ranker/base.py

```diff
@@ -13,29 +14,31 @@ def get_dictionary_term_list(term,dictionary=DICTIONARY_PATH) :
 					term_list[doc_data_parts[0]] = doc_data_parts[1].replace("\n","")
 	return term_list
 
-def get_number_of_docs(dictionary):
-	with open(dictionary) as f:
-		for line in f:
-			if 'num_documents' in line:
-				return int(line[len('num_documents')+1:])
+def get_number_of_docs(dictionary_info):
```

Comment:
Maybe rename dictionary_info to something indicating, that this is a filepath/filename?